src/cascadia/CascadiaPackage/Package.appxmanifest

```diff
@@ -60,6 +61,12 @@
             <desktop:ExecutionAlias Alias="wt.exe" />
           </uap3:AppExecutionAlias>
         </uap3:Extension>
+        <uap5:Extension Category="windows.startupTask">
+          <uap5:StartupTask
+            TaskId="StartTerminalOnLoginTask"

```

Comment:
Should we make this a GUID or something more ID-like?

Reply:
It's a fair question. The TaskId is scoped to the UWP package, so there's no danger of naming collisions from external sources. Because the total number of Tasks that might be added in the future would be < 5 (one for each application activation type IIRC), and because that is not something that would happen often, I think having the taskID be a human-recognizable string adds maintainability.

That said, I'm totally open to changing it to a GUID :)

Reply:
In that case, I don't have an issue with something human readable. Thanks! :)